src/SpiceSharpParser/Common/Mathematics/Probability/CustomRandomNumberProvider.cs

```diff
@@ -18,7 +18,7 @@ public CustomRandomNumberProvider(Pdf pdf, IRandomDoubleProvider baseRandom)
             _baseRandom = baseRandom;
         }
 
-        public double NextDouble()
+        public double NextSignedDouble()

```

Comment:
What is this function _supposed_ to return? I can see that it gives a value bound by `int.MaxValue` but what's the ideal behavior?

Reply:
@svenboulanger `Next` should return random integer from 0 to int.max. 
I'm aware that current implementation `int.MaxValue * NextDouble()` is very far away from perfect.

Reply:
I'm not saying it's far from perfect. It just seems pretty arbitrary to use `int.MaxValue`. Why would this need to be `int.MaxValue` and not a double between 0 and 1? Why not `double.MaxValue`? What is it going to be used for?

Reply:
@svenboulanger We are talking about `Next` method that is also in standard `Random` class. 
I wanted to have something similar in custom random provider. If it doesn't make sense, I will remove it :)